store-redis/settings.gradle

```diff
@@ -8,5 +8,4 @@ pluginManagement {
     maven { url "https://jitpack.io" }
   }
 }
-
-rootProject.name = 'nats'
+rootProject.name = "store-redis"
```

Comment:
Just to be super sure because I'm a bit confused: Moving the nats `settings.gradle` to `store-redis/settings.gradle`, and renaming the project to `store-redis`?

Reply:
Nevermind, it's just how it's displaying here. Sorry